### ISSUE TITLE
Create a list of static pages inside theme customize section

### ISSUE THREAD

**Issue by Brun0AFK** [rainlab/pages-plugin#243]
Hello, 

I trying to figure out how to list all static pages in dropdown inside theme customization, and inside static page variable. Is this even possible? I want that when someone select a page from that dropdnow, I can grab that page url or title. Any advice?

**Comment by jan-vince** [comment]
Probably not the best way, but this is how a access Pages list from php part of layout:

```php
==
<?php
function onInit(){

$theme = Cms\Classes\Theme::getEditTheme();
$pageList = new RainLab\Pages\Classes\PageList($theme);

$pages = $pageList->listPages();

// send pages list to twig
$this['pagesList'] = $pages;

// or iterate
foreach($pages as $page) {
  // do what you need
}

?>
==
```

**Comment by LukeTowers** [comment]
Fixed with #259
